py-scripts/lf_interop_qos.py

```diff
@@ -859,6 +864,11 @@ def monitor(self):
                         self.df_for_webui.append(self.overall[-1])
                         previous_time = current_time
             if self.dowebgui == "True":
+                for key,value in t_response.items():
+                    row_data = [value[0],value[1]]
+                    individual_device_data[key].loc[len(individual_device_data[key])] = row_data
```

Comment:
Not requesting changes at this time, but this is basically as complex of a line as I'm comfortable with (would prefer less complex but okay as is I suppose).